libprecompiled/extension/WedprPrecompiled.cpp

```diff
@@ -41,6 +41,12 @@ using namespace dev::storage;
 using namespace dev::precompiled;
 
 // confidential payment
+const char CONFIDENTIAL_PAYMENT_VERSION[] = "v0.2-generic";
+const string CONFIDENTIAL_PAYMENT_REGEX_WHITELIST = "^v0.2-generic$";

```

Comment:
Is it possible to use const char CONFIDENTIAL_PAYMENT_REGEX_WHITELIST[] ?

Reply:
done